### ISSUE TITLE
Elevated Mode not launching

### ISSUE THREAD

**Issue by AndrewM521** [elvissteinjr/DesktopPlus#8]
Elevated Mode is not auto-starting with SteamVR and ONLY elevated mode. For some reason when I try to elevate Desktop+ as admin with the bat file nothing actually happens, so I tried manually setting Desktop+ as admin and that's when it stopped launching on startup

**Comment by elvissteinjr** [comment]
I recommend taking a second look at the readme in regards how elevated mode is supposed to work. As it's intended, elevated mode is not supposed to be a permanent state. There are theoretical security concerns with it, so general recommendation is to switch to it, do whatever has to be done with higher privileges, and then go back to normal mode.
What the bat file does is adding a scheduled task allowing elevated launch without UAC prompt. This is used to launch in elevated mode from the Troubleshooting section in the settings interface of Desktop+.

This doesn't stop you from going permanent elevated mode if you really wish to do so, but that's unrelated to what the bat file adds to the system.
Regarding auto-launch breaking: If you force launching as admin on Desktop+' executable, then no program can just launch it anymore unless they have the same privilege level. That's just how it is.
If you really do need SteamVR to launch it in elevated mode, either run SteamVR as admin as well or point Desktop+' vrmanifest file to run the scheduled task instead. I recommend doing neither, personally.

**Comment by AndrewM521** [comment]
Thank you for responding so quickly. The only reason I have it in elevated mode is that I have discord as admin and my VR station is in another room from my pc so I don't want to keep walking from room to room when I need to do something with discord. I did read the security implications that elevated mode creates, and the only 3rd party apps for VR I have are the advanced settings and I likely won't be getting any other 3rd party app (other than this because I really enjoy it). I will try a full reinstall of Desktop+ just to make sure it's not something with the files that I already have. Again that you for responding so quickly and I really enjoy Desktop+ from the time I have spent with it already.
